uPortal-webapp/src/main/resources/layout/theme/respondr/navigation.xsl

```diff
@@ -89,6 +89,54 @@
       <nav class="portal-nav">
         <div id="sidebar" class="sidebar-offcanvas container-fluid">
             <div id="portalNavigation" class="fl-widget" aria-selected="false">
+                <!-- Optional Tab Groups -->
+                <xsl:if test="$USE_TAB_GROUPS='true'">
+                    <!--
+                    <span id="activeTabGroup" style="display:none;"><xsl:value-of select="/layout/navigation/tabGroupsList/@activeTabGroup"/></span>
+                    <span id="useTabGroup" style="display:none;"><xsl:value-of select="$USE_TAB_GROUPS"/></span>
+                    <h3 class="fl-offScreen-hidden">Tab Groups</h3>
+                    -->
+                    <div id="portalNavigationTabGroup" class="fl-widget-inner header">
+                        <!-- <span id="activeTabGroup" style="display:none;"><xsl:value-of select="/layout/navigation/tabGroupsList/@activeTabGroup"/></span> -->
+                        <ul id="portalNavigationTabGroupsList" class="menu fl-tabs list-group list-group-horizontal">
+                            <xsl:for-each select="tabGroupsList/tabGroup">
+                                <xsl:variable name="TABGROUP_POSITION"> <!-- Determine the position of the navigation option within the whole navigation list and add css hooks for the first and last positions. -->
+                                    <xsl:choose>
+                                        <xsl:when test="position()=1 and position()=last()">single singleTabGroup</xsl:when>
+                                        <xsl:when test="position()=1">first</xsl:when>
+                                        <xsl:when test="position()=last()">last</xsl:when>
+                                        <xsl:otherwise></xsl:otherwise>
```

Comment:
Is this CSS we control?
Could CSS [`:first-child`](https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child) and [`:last-child`](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child) be used?

Reply:
In theory

```css
.portal-navigation-tabgroup:first-child {} /* in lieu of `.portal-navigation-tabgroup.first` */
.portal-navigation-tabgroup:last-child  {} /* in lieu of `.portal-navigation-tabgroup.last` */
.portal-navigation-tabgroup:only-child  {} /* in lieu of `.portal-navigation-tabgroup.singleTabGroup` */
```

could be used

Reply:
looks like it'll work.